deployment/deployment_test.go

```diff
@@ -115,10 +115,8 @@ func TestCreate(t *testing.T) {
   "data": {
     "createDeployment": {
       "id": "ckbv818oa00r107606ywhoqtw",
-      "config": {
-        "executor": "CeleryExecutor"
-      },
-      "urls": [
+			"executor": "CeleryExecutor",
```

Comment:
Is this the correct indention?  Did the linter do this too?

Reply:
Not sure why it looks different on the github UI. It looks fine on VS.